registry/repositories.go

```diff
@@ -105,8 +106,8 @@ func (registry *Registry) StreamRepositories(ctx context.Context) (<-chan string
 }
 
 func (registry *Registry) tryFallback(ctx context.Context, regChan chan string, errChan chan error) error {
-	regurl := registry.url("/api/v0/repositories/")
-	registry.Logf("attempting DTR fallback at %v", regurl)
+	regurl := string(registry.url("/api/v0/repositories/"))
+	registry.Logf("Attempting DTR fallback at %v", regurl)
```

Comment:
While it's a good practice for log messages to start with capitals, the rest of the module uses lower-case initial characters for log messages. If you'd like to raise a separate PR to update all of the log messages, that would be welcome.